client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -67,12 +84,12 @@ const ProfileInfoEdit = () => {
 
           <Form.Group controlId="formProfilePic">
             <Image
-              src={profilePic}
+              ref={uploadedImage}
               roundedCircle
               height="184px"
               width="184px"
             />
-            <Form.Control type="file" />
+            <Form.Control type="file" accept="image/*" onChange={handleImageUpload} multiple="false" />
```

Comment:
Sorry for some reason this comment was left out (I blame GitHub)
Currently It does allow selecting multiple files:
![Screen Shot 2021-09-06 at 9 00 13 pm](https://user-images.githubusercontent.com/63642262/132207703-652a87ce-0f7b-4f06-8ad9-905a55b00a33.png)

We can change it to `multiple={false}` to fix it, I think it expects a boolean instead of a string. Also, fyi a string is treated as `true` in JS, and that's why it's allowing multiple files.

Reply:
Thanks that works